docs/features.md

```diff
@@ -388,38 +374,51 @@ For more information check the [stubs settings section](../settings.md#stub-gene
+| cf       | cycle-fn-literal        | Cycle fn literal `(fn []), #()`        | `[file-uri,row,col]`                             | ✅                         | <img src="../images/features/cycle-fn-literal.gif" width=480>        |
+| cn       | clean-ns                | Clean namespace                        | `[file-uri,row,col]`                             | ✅                         | <img src="../images/features/clean-ns.gif" width=360>                |
+| cp       | cycle-privacy           | Cycle privacy of def/defn              | `[file-uri,row,col]`                             | ✅                         |                                                                      |
+| ct       | create-test             | Create test                            | `[file-uri,row,col]`                             | ✅                         |                                                                      |
+| ef       | extract-function        | Extract Function                       | `[file-uri,row,col,name]`                        | ✅                         | <img src="../images/features/extract-function.gif" width=360>        |
+| el       | expand-let              | Expand let                             | `[file-uri,row,col]`                             |                           |                                                                      |
+| fe       | create-function         | Create function from example           | `[file-uri,row,col]`                             | ✅                         | <img src="../images/features/create-private-function.gif" width=360> |
+| il       | introduce-let           | Introduce let                          | `[file-uri,row,col,name]`                        |                           |                                                                      |
+| is       | inline-symbol           | Inline Symbol                          | `[file-uri,row,col]`                             | ✅                         | <img src="../images/features/inline-symbol.gif" width=360>           |
+| ma       | resolve-macro-as        | Resolve macro as                       | `[file-uri,row,col]`                             | ✅                         | <img src="../images/features/resolve-macro-as.gif" width=720>        |
+| md       | move-coll-entry-down    | Move coll entry down                   | `[file-uri,row,col]`                             | ✅                         | <img src="../images/features/move-coll-entry.gif" width=720>         |
+| mf       | move-form               | Move form                              | `[file-uri,row,col,filename]`                    | ✅                         |                                                                      |
```

Comment:
I was wondering, maybe we could receive `file-uri, row, col, dest-namespace`, that would be a better UX?
Also, I though about not receiving that at all, and asking it so a future code action could use the same command, similar to how we do with `resolve-macro-as` and `create-test`, WDYT?

Reply:
I used the dest-filename because editors can autocomplete it very easily.

Reply:
I think receiving a `ns` has a better semantic for the command as it doesn't make sense to move to a file outside the project, or not in the source-path?
We do similar on `ns` renames, we ask for the new ns name and not the filename.
I know editors can complete filenames, but as a IDE maybe we should restrict the input to make even clear what we expect to user input

Reply:
I think ns has a ton of ambiguity that files don't.
Macros in cljs often have two files with the same namespace, one .clj file one .cljs file.
If we want to extend this to create a new ns, you'd still have to ask for the path.
Rename can assume the same src folder, whereas this would often be used to move from .clj to a shared .cljc src dir.

If you wanted, you can add a restriction that checks that the filename is within one of the source paths.

Reply:
I see, makes sense I think